### ISSUE TITLE
Textures not showing on iPhone

### ISSUE THREAD

**Issue by rich-gg** [processing/p5.js#3671]
#### Nature of issue?

- [X ] Found a bug
- [ ] Existing feature enhancement
- [ ] New feature request

#### Most appropriate sub-area of p5.js?

- [ ] Color
- [ ] Core/Environment/Rendering
- [ ] Data
- [ ] Events
- [ ] Image
- [ ] IO
- [ ] Math
- [ ] Typography
- [x] Utilities
- [X ] WebGL
- [ ] Other (specify if possible)

#### Which platform were you using when you encountered this?

- [ X] Mobile/Tablet (touch devices)
- [ ] Desktop/Laptop
- [ ] Others (specify if possible)

#### Details about the bug: 

Hi there

I've been working a year on my p5.js website and last week I released it
I don't own Apple products so I could test on them so what I did was use the chrome 'sort of' emulator and I had to rely on facebook friends for some beta test. People replied my site worked ok

Little did I know that the iPhone users didn't see almost anything and yet were content with my site :(

Today I hear that my textures (meaning all the images of my WebGL site) are just not showing on iPhones

**update**
I think I would go for a Texture() failure of p5.js, some WebGL problem with some iPhones

Feedback tells me the site is ok on : 
- iPhone 6S 12.1.4
- iPhone SE 12.1.4 

Textures not showing on : 
- iPhone 8+ 12.1.4
- iPhone X 12.1.4
- iPhone XS 12.1.4

Same behaviour on Safari and Chrome

Would you try it out ?
https://rich.gg

Would you know how to fix that ?

thank you

ps: In 20 years I have had two major blows in my professional work and, beside loosing everything twice, both of them have been caused by Apple, its unwavering will to be uncompatible and to charge the maximum to use their system. And this is one of them.

**Comment by Spongman** [comment]
works on iPhone 6S 12.1.4

**Comment by stalgiag** [comment]
Perhaps related to #3293

This is tough to debug without the hardware. I am going to try to borrow an iphone soon but in the meantime, do you know if there is any console output for any of the failing devices?

**Comment by rich-gg** [comment]
it really look like this is the same bug
In righ.gg I temporarely changed the geometry on my 3D panels 
to use the work around

```
beginShape();
vertex(-larg, -alt, 0, 0);
vertex(larg, -alt, 1, 0);
vertex(larg, alt, 1, 1);
vertex(-larg, alt, 0, 1);
endShape();
```
hope it works for my users

Unfortunatly I don't have access to any iPhone 8, 9 or X

**Comment by stalgiag** [comment]
Cool thanks for the update! So it seems that `plane()` geometry isn't texturing correctly on a specific set of iPhone hardware. Very odd 😕

**Comment by rich-gg** [comment]
Still cannot tell if the fix worked
But one thing I'd like to share with you is that 
when I started playing around with p5.js in WebGL,
2 or 3 years ago, I had to use the shape trick to
Have my textures showing.
That is why I was not  surprised by the work around.

Cannot remember if that was on PC or Android but,
By then I'm sure the problem was not iPhone only

Don't know if it'll help but I thought "every details
Might count in this investigation" ;)

**Comment by rich-gg** [comment]
Nope, it doesn't work :(

**Comment by stalgiag** [comment]
Is someone able to test to see if this problem is also fixed by #3755? This seems likely.

**Comment by rich-gg** [comment]
@stalgiag 
I can have someone test it but I need to know if the fix is in 
https://cdnjs.cloudflare.com/ajax/libs/p5.js/0.7.3/p5.min.js

if not just give me the proper url and I'll test it

**Comment by stalgiag** [comment]
Hi @rich-gg I replied to this question [in this thread](https://github.com/processing/p5.js/issues/3518#issuecomment-494492879)

**Comment by stalgiag** [comment]
I believe this is closed via #3755. See the discussion at the bottom of #3518 for confirmation.
